CONTRIBUTING.adoc

```diff
@@ -451,13 +451,43 @@ Asciidoc file, I would then create the file `content/my-clean-url/index.adoc`.
 (Advanced Haml users would create `content/my-clean-url/index.html.haml`).
 
 
-== Deploying on GitHub pages
+== Deploying jenkins.io on GitHub pages
 
-There is rudimentary (as in if it doesn't work, you're on your own) support for
-publishing the static site to
+There is rudimentary support for publishing the `jenkins.io` static site to
 link:https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages[GitHub Pages]
-for demonstration purposes.
-
-Run `make user-site GITHUB_USER=<your_username>` to push the site content
-`origin/gh-pages`.  It can then  so they can be viewed at
-`https://your_username.github.io/jenkins.io`.
+for demonstration purposes. "Rudimentary support" means that any images or
+internal links that use _absolute_ as opposed to _relative_ references will
+appear broken/not found once published to GitHub pages. Also, if anything else
+doesn't work, you're on your own.
+
+This feature is useful if you are a contributor and wish to demonstrate what
+your changes will look like to others, without the reader needing to generate
+their own version of the static site locally (via `make run`).
+
+Assuming you're updating `jenkins.io` content in a branch of your `jenkins.io`
+fork, a recommended approach would be to conduct the following:
+
+. Ensure you have configured GitHub pages (with a `gh-pages` branch of your
+  `jenkins.io` fork). This should be in both your local clone and GitHub fork of
+  `jenkins.io` (i.e. your `origin`).
+. Create a new branch from `master` (e.g. called `demosite`) to deploy your
```

Comment:
@gilesgas Most of this is only needed right now.  Once the PR goes through it won't be needed.